doc/plugin_server_nodeattestor_awsiid.md

```diff
@@ -0,0 +1,13 @@
+# Server plugin: NodeAttestor "aws_iid"
```

Comment:
Should the name of these files also be `aws_iid` instead of `awsiid`?

Reply:
done